integration/reconfiguration_test.go

```diff
@@ -163,6 +163,42 @@ func TestReconfiguration_ExplicitStreams(t *testing.T) {
 	doReconfigurationTest(t, state, input, output)
 }
 
+func TestReconfiguration_StreamFileDoesNotExist(t *testing.T) {
```

Comment:
Which stream file does not exist? You are conflating two separate code paths here, which means one is not being tested. There are two different flags, so we need two have two disjoint tests. (A table-driven test can be sufficient here, where the test parameters are the flag name and the expected error message.)